onboarding/engineering-introduction.md

```diff
@@ -1,46 +1,28 @@
 ---
-title: Onboarding into Engineering
-description: Your first steps to being productive
+title: Engineering Team Introduction
+description: A brief introduction
 ---
 
-### Onboarding
```

Comment:
@orta I added a couple of things here, but mostly just moved things around slightly.